### ISSUE TITLE
Magazine decrypted is not the same as what was downloaded on the tablet

### ISSUE THREAD

**Issue by MindController** [apprenticeharper/DeDRM_tools#1007]
I download a copy of the DRM'd magazine and view it in the kindle app.  It shows the full magazine.  This version looks just like the print version.  

But when I use the backup.ab file and copy over the magazine from the tablet (it's in prc) format, calibre along with DeDRM views it as a mobi file and gives me this terrible compressed version that looks nothing like the actual magazine.

Is there anyway around that?

**Comment by ElleKayEm** [comment]
Have you tried looking at the DRM-free version in something besides the calibre ebook viewer?

**Comment by MindController** [comment]
Yes.  The file it default converts it too is a MOBI file.  And that is the shrunk down version.  Even outside the calibre ebook viewer.

**Comment by ElleKayEm** [comment]
Hmmm, I wonder if the tablet version has higher res images stored separately from the basic book file.  Because PRC and MOBI are both Mobipocket files.
